### PR TITLE
Use the hardware set specific nfs url

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by toabctl** [comment]
+1

File: scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-2a-sbd-kvm.yaml [hunk 1]
```diff
@@ -89,7 +89,7 @@ proposals:
     ha:
       storage:
         shared:
-          device: 10.162.26.129:/var/qa2/ha-database
+          device: ##shared_nfs_for_database##
```
Comment (toabctl) [inline scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-2a-sbd-kvm.yaml]: How is this string replaced?

Comment (dirkmueller) [inline scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-2a-sbd-kvm.yaml]: it is currently hardcoded in the jenkins job scenario file. To be clarified and unified by the card in team green about cleaning up the qa jobs and unifying them with JJB

**Comment by vuntz** [comment]
+1

**Comment by toabctl** [comment]
+1

File: scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-2a-sbd-kvm.yaml (hunk 1 above)
Comment (dirkmueller) [inline scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-2a-sbd-kvm.yaml]: https://github.com/SUSE-Cloud/automation/blob/jenkins/scripts/jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-2a.xml#L172

**Pull request merged**